images/origin/scripts/recycler.sh

```diff
@@ -22,8 +21,20 @@ if [[ ! -d ${dir} ]]; then
     exit 1
 fi
 
+# remount to nfsv3 so not restricted by nfs v4 acls, see nfs4_acl(5)
```

Comment:
is nfsv3 always available? remounting to escape acls seems bad (to allow, and to depend on)... what prevents other pods from doing this?